examples/example.py

```diff
@@ -19,14 +19,13 @@ def main():
     # Enable default logger for logging message
     core.enable_default_logger()
 
+    # The Main object handles the calling of Decanter's API.
+    client = core.CoreClient()
```

Comment:
no need to add create method, just supply username, password, host as construct parameter here, and connect and create context in class constructor